packages/lib-react-components/src/helpers/getGradientShade/getGradientShade.spec.js

```diff
@@ -0,0 +1,13 @@
+import getGradientShade from './getGradientShade'
+
+describe('Helper > getGradientShade', function () {
+  it('should be a function', function () {
+    expect(getGradientShade).to.be.a('function')
+  })
+
+  it('should return the expected results', function () {
```

Comment:
All tests should return the expected results. Can you describe what this one does here?